backend/tests/test_002_user.py

```diff
@@ -272,12 +276,13 @@ async def test_update_user_password(
         assert res['user']['email'] == user.email
 
     @pytest.mark.asyncio
-    async def test_invite_user_with_by_admin(
+    async def test_invite_user_by_super_admin(
         self, app: FastAPI, session: Session, client: AsyncClient
     ) -> None:
         user_payload = {
-            "fullname": "Invited User",
-            "email": "invited_user@akvo.org",
+            "fullname": "Invited Admin",
+            "email": "admin@akvo.org",
+            "role": UserRole.admin.value,
```

Comment:
business unit is required when it's admin, editor, or viewer

Reply:
I have pushed an update for this